android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1730,6 +1730,18 @@ public static boolean updateSelectionRange(KeyboardType kbType, int selStart, in
     return result;
   }
 
+  public static void resetContext(KeyboardType kbType) {
+    if (kbType == KeyboardType.KEYBOARD_TYPE_INAPP) {
+      if (InAppKeyboard != null && InAppKeyboardLoaded && !InAppKeyboardShouldIgnoreTextChange) {
```

Comment:
Why is this testing `InAppKeyboardShouldIgnoreTextChange` but the other one tests `SystemKeyboardShouldIgnoreSelectionChange`? Do those two variables serve the same purpose?

Reply:
Oops, copy-paste error. I think I should use `SystemKeyboardShouldIgnoreTextChange`